DOCS-526: ifNotFinite docs. Ternary operator RU translation (#9650)
ClickHouse/ClickHouse: docs/en/query_language/functions/conditional_functions.md

```
Note: `NULL` values are not used in this example, check [NULL values in conditionals](#null-values-in-conditionals) section.

> ## Ternary operator {#ternary-operator}

Reviewer: ```suggestion
## Ternary Operator {#ternary-operator}
```

It works same as `if` function.

Syntax: `cond ? then : else`

> Returns `then` if the `cond` is truthy (greater than zero), otherwise returns `else`.

Reviewer: ```suggestion
Returns `then` if the `cond` evaluates to be true (greater than zero), otherwise returns `else`.
```
